README.md

```diff
@@ -167,24 +194,25 @@ The monkey can check to see if the application is progressing. It does this by u
 The fingerprint function is supplied by the client. One handy, free fingerprint function is `elementAccessorDump()` found in the open source [Tuneup.js](https://github.com/alexvollmer/). This function creates a logical textual description of the main view.
 
 
+
 ```
 //ANR handling
 #import ./tuneup.js
 ...
 var aFingerprintFunction = function() {
-    var mainWindow = UIATarget.localTarget().frontMostApp().mainWindow();
-    //if an error occurs log it and make it the fingerprint
-    try {
-        var aString = mainWindow.elementAccessorDump("tree", true);
-        if (monkey.config.anrSettings.debug) {
-            UIALogger.logDebug("fingerprintFunction tree=" + aString);
-        }
-    }
-    catch (e) {
-        aString = "fingerprintFunction error:" + e;
-        UIALogger.logWarning(aString);
-    }
-    return aString;
+	var mainWindow = UIATarget.localTarget().frontMostApp().mainWindow();
```

Comment:
Again, can you double check the white space introduced here? I'd like to reduce the diff of this PR down to necessary changes if possible.